sbncode/TPCReco/VertexStub/CMakeLists.txt

```diff
@@ -1,3 +1,30 @@
+art_make_library(LIBRARY_NAME sbncode_StubHelpers
```

Comment:
I'm not sure what the alternative is here. It's a library that contains a few .cxx files, I think if the name is any one in particular then it is confusing.

I think it's good that the library name is explicit. This way you can look at the CMake file and see what the library is, instead of relying on a convention that is only sometimes followed in larsoft.

Reply:
This was an attempt to make that convention more followed. LArSoft almost universally follows it (exceptions are a handful). ICARUS as well (that probably just proves that I really think it's a good practice...). SBND somehow less.

The alternative is conventional, to use the name of the directory. The compromise to pick a name is to put the few `.cxx` files in their own subdirectory (e.g. `sbncode/TPCReco/VertexStub/Helpers`).

Strangely enough, the meaningfulness of the library name has not been that paramount in my experience: that name only enters my life when I have written the source code and I need the names of the libraries connected to the headers I included. The most automatic the conversion header → library name is, the better for me. Maybe your experience is different?
One more thing before I stop rambling: again from my experience, following a convention makes life of maintainers much, much easier.

About "can look at CMake file": CMake files are not distributed with the code, and to look at it one needs to check out the repository or use web browsing (admittedly, GitHub makes the latter easy enough).

Reply:
Ok, that makes sense. I will change it to use the conventional library name.